client/src/components/pages/Home/index.js

```diff
@@ -1,14 +1,138 @@
-import React from 'react';
+import React, { Component } from 'react';
+import Input from '../../common/Input';
+import Avatar from '../../common/Avatar';
+import UserMenu from '../../common/UserMenu';
+import People from './People';
+import Messages from './Messages';
+import api from '../../../services/api';
+import { filter } from '../../assets/svgPaths';
 import './index.css';
-import UnderConstruction from '../UnderConstruction';
-
-const Home = () => {
-  return (
-    <>
-      Home
-      <UnderConstruction />
-    </>
-  );
-};
+
+class Home extends Component {
+  state = {
+    data: {},
+    search: '',
+    menu: false,
+    messages: [
```

Comment:
again, don't like having testing values mixed with code like this very much. You'll forget to remove them, it's polluting the production code and can be confusing for other devs.. Better to have it in a separate file, you could make the API return this fake data for example